core/standalone/src/main/resources/logback-standalone.xml

```diff
@@ -29,6 +29,8 @@
 
     <!-- Kafka -->
     <logger name="org.apache.kafka" level="ERROR" />
```

Comment:
I guess the existing definition for `org.apache.kafka` is for Kafka clients (consumer, producer)?

Reply:
Yes